### PR TITLE
Aastha_Assignment_26_Done_35mins

### PR DESCRIPTION
**Pull request by AasthaJJun20**
-> pushed class SumAllNumbersInString for assignment-26
->Took approx 35 mins to complete the assignment
-> PFB screenshot:
SumAllNumbersInString-output

### CODE REVIEW

File: src/aasthaJJun20/SumAllNumbersInString.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+				temp += String.valueOf(input.charAt(index));
+			} 
+			else {
+				if (!temp.equals("")) {
+					sum += Integer.parseInt(temp);
+					temp = "";
+				}
+			}
+		}
+		sum = (!temp.equals("")) ? (sum + Integer.parseInt(temp)) : sum;
```
Comment (NikhilMJun20) [inline src/aasthaJJun20/SumAllNumbersInString.java:27]: Good use of ternary operator!

**Review by NikhilMJun20 — APPROVED** [review]
Code looks good .
I am approving the PR.

File: src/aasthaJJun20/SumAllNumbersInString.java (hunk 1 above)
Comment (AasthaJJun20) [reply src/aasthaJJun20/SumAllNumbersInString.java:27]: Thanks Nikhil!

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks amazing to me. Doing excellent.

**Pull request merged**